README.md

```diff
@@ -115,16 +115,35 @@ FROM plone/plone-backend:6.0.0b2
 RUN ./bin/pip install "relstorage==3.4.5" "psycopg2==2.9.3 --use-deprecated legacy-resolver"
 ```
 
-Also create a `requirements.txt` file, with packages to be installed:
+Make sure any Plone add-ons you are adding via `pip install` have their corresponding entry
+points declared in `setup.py` — otherwise they will not be loaded by Plone:
 
-```plain
-pas.plugin.authomatic
+```
+...
+    entry_points="""
+    [z3c.autoinclude.plugin]
+    target = plone
+    """,
+...
+
+```
+
+Also make sure that your add-ons do not use `zc3.autoinclude` in their `configure.zcml`
+configuration file.  Instead, list the packages to include in your add-on's
+`configure.zcml`, explicitly:
+
+```xml
+...
+  <!-- -*- include dependencies / sync me with setup.py -*- -->
+  <include package="z3c.jbot" />
+...
 ```
 
-Build your new image
+Now build your new image:
 
 ```shell
 docker build . -t myproject:latest -f Dockerfile
+# buildah build works too
```

Comment:
Huh? Are you from Boston, where you pahk ya cah in da cah pahk?

Seriously, though, what is this comment referring to?

Otherwise this LTGM, but I should not be the sole reviewer, even though it is "just docs".

Reply:
`buildah build` is a `docker build` replacement that does not need root.  I use `buildah` exclusively.

Reply:
If we introduce a new term in any documentation, even a comment in a readme, then we need to provide sufficient explanation or a link to further information. I had never heard of `buildah` until now, but it must be awesome because it has a dog in its logo 🐶 .

Reply:
Fair.  I would not mind if that line was removed from the PR.